### PR TITLE
Add internal ESLint rule `await-cli-tests`

### PR DESCRIPTION
**Pull request by fisker**
## Description

Prevent mistakes.

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: scripts/tools/eslint-plugin-prettier-internal-rules/no-multiple-cli-tests.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+].join("");
+
+const MESSAGE_ID_CALL = "no-multiple-cli-tests/call";
+const MESSAGE_ID_GETTER = "no-multiple-cli-tests/getter";
+
+module.exports = {
+  meta: {
+    type: "suggestion",
+    docs: {
+      url: "https://github.com/prettier/prettier/blob/main/scripts/tools/eslint-plugin-prettier-internal-rules/no-identifier-n.js",
```
Comment (sosukesuzuki) [inline scripts/tools/eslint-plugin-prettier-internal-rules/no-multiple-cli-tests.js:16]:
```suggestion
      url: "https://github.com/prettier/prettier/blob/main/scripts/tools/eslint-plugin-prettier-internal-rules/no-multiple-cli-tests.js",
```

**Review by sosukesuzuki — COMMENTED** [review]
The name `no-multiple-cli-tests` is confusing for me. What do you think about `no-awaited-cli-tests`?

**Comment by fisker** [comment]
Sure.

**Comment by fisker** [comment]
You mean `no-non-awaited-cli-tests`?

**Comment by sosukesuzuki** [comment]
> You mean no-non-awaited-cli-tests?

Yes! Sorry

**Comment by fisker** [comment]
Or maybe just `await-cli-tests`?

**Review by duailibe — APPROVED** [review]

**Review by sosukesuzuki — APPROVED** [review]

**Pull request merged**